profiles/ding2/modules/ding_adgangsplatformen/ding_adgangsplatformen.module

```diff
@@ -296,6 +297,11 @@ function ding_adgangsplatformen_generate_login_url($destination = '') {
     $authorization_url .= '&idp=' . $_REQUEST['idp'];
   }
 
+  // Filter on agency type (only show "folkebibliotekeren).
+  if ($configuration['agencytype']) {
+    $authorization_url .= '&agencytype=folk';
+  }
```

Comment:
This definitely works, but using a boolean setting for this seems a bit weird (when looking only at the code). Something like

```php
if ($configuration['agencytype']) {
  $authorization_url .= '&agencytype=' . urlencode($configuration['agencytype']);
}
```

would make a little more sense.

Reply:
Not sure i understand... it has to be `agencytype=folk` and `folk` is the only supported value by DBC